tests/test_parsed_age.py

```diff
@@ -0,0 +1,7 @@
+from src.parsed_age import parsed_age
+
+
+def test_parsed_age():
+    assert parsed_age("8 years old") == 8
+    assert parsed_age("2 years old") == 2
+    assert parsed_age("9 years old") == 9
```

Comment:
Add `19` as a test. Add `101` as a test. How about `1,000` or `1000` . What ages should it parse? The age of the earth?

Reply:
The original code challenge was parsing the age of a young girl between the age of 0-9 that is why it doesn't take a value higher than a single integer.